packages/@aws-cdk/aws-codepipeline/test/test.cloudformation-pipeline-actions.ts

```diff
@@ -62,7 +62,8 @@ export = {
     stage: prodStage,
     stackName,
     changeSetName,
-    role: changeSetExecRole,
+    runOrder: 321,
```

Comment:
Super minor, but why add this `runOrder` here? Seems you can safely test this in a different place, like the new test you added below.